packages/next-dashboard/src/components/charts/miniCharts/RenderCustomizedPieLabel.js

```diff
@@ -32,25 +30,21 @@ const RenderCustomizedPieLabel = <T: Plot>({
   fill,
   value,
   width,
-  height,
   valueFormatter,
   payload,
 }: Props<T>) => {
-  const labelRadius = outerRadius * 1.25;
+  let labelRadius = outerRadius * 1.35;
+  if (width > 400) {
+    labelRadius = outerRadius * 1.25;
+  }
```

Comment:
Spontaneously wouldn't be easier to read and produce more consistent results? As in, I assume this is to adjust for scaling on smaller radiuses, where I think a constant offset would be more beneficial than a bigger multiplier. But that's 
```
  let labelRadius = outerRadius * 1.25;
  if (width <= 400) {
    labelRadius += CONSTANT_OFFSET;
  }
```

Alternatively, to maintain identical output

```
  let labelRadius = outerRadius * 1.25;
  if (width <= 400) {
    labelRadius *= 1.08;
  }
```

Not really important just personal preference.

Reply:
Oh yeah, that might be a better Idea, I'll test using what you're suggesting 👍